src/txmempool.cpp

```diff
@@ -1095,7 +1318,18 @@ void CTxMemPool::check(const CCoinsViewCache *pcoins) const
             nFeesCheck += ancestorIt->GetModifiedFee();
             nSigOpCheck += ancestorIt->GetSigOpCount();
         }
+        if (it->GetCountWithAncestors() != nCountCheck)
+        {
+            fprintf(stderr, "nsizewith ancestors is %ld countcheck is %ld\n", it->GetCountWithAncestors(), nCountCheck);
 
+            setEntries parents = GetMemPoolParents(it);
+            int nParent = 0;
+            for (txiter parent : parents)
+            {
+                nParent++;
+                fprintf(stderr, "parent %d size ancestors %ld\n", nParent, parent->GetCountWithAncestors());
+            }
+        }
```

Comment:
Is this something added for testing purpose?

Reply:
just some test code I left in, taken out now..thx

Reply:
ok